src/test/resources/groups.xml

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="GroupsTestNGHomework7" verbose="1" >
+
+    <test name="CuraHealthPageTests">
+        <groups>
+            <run>
+                <include name="login"  />
```

Comment:
You have included all of your groups in this file. You kind of already defined this inside suite as it's running all the test cases there. The power of groups come when you select only specific tests to be run while maybe looking in all your test classes. Use also exclude instead of include for some groups to achieve that. Thanks!

Reply:
done